get-started/codelab/index.md

```diff
@@ -0,0 +1,1002 @@
+---
+layout: tutorial
+title: Write Your First Flutter App
+permalink: /get-started/codelab/
+---
+
+<figure class="right-figure" style="max-width: 250px">
+    <img src="images/startup-namer-app.gif"
+         alt="Animated GIF of the app that you will be building."
```

Comment:
this is not good alternative text. See https://html.spec.whatwg.org/multipage/images.html#alt for advice.